engine/src/main/java/org/terasology/rendering/dag/FinalPostProcessingNode.java

```diff
@@ -143,14 +145,14 @@ private void renderFinalMonoImage() {
     // TODO: have a flag to invert the eyes (Cross Eye 3D), as mentioned in
     // TODO: http://forum.terasology.org/threads/happy-coding.1018/#post-11264
     private void renderFinalStereoImage(RenderingStage renderingStage) {
-        if (postProcessor.isNotTakingScreenshot()) {
-            sceneFinal.bind();
-        } else {
-            ocUndistorted.bind();
-        }
-
         switch (renderingStage) {
             case LEFT_EYE:
+                if (postProcessor.isNotTakingScreenshot()) { // TODO: verify if this works
```

Comment:
It just renders two images side-by-side allegedly from slightly different point of views and appropriately distorted for the lenses in the Oculus headset. It won't break your monitor. =)